tools/ci_build/github/azure-pipelines/templates/windows-build-tools-setup-steps.yml

```diff
@@ -26,19 +26,25 @@ steps:
     #     downloadDirectory: '$(Build.BinariesDirectory)\python'
 
     # Temporary bypass of artifacts permission issue
-    - task: PowerShell@2
+    - task: CmdLine@1
+      displayName: 'Download azcopy'
+      inputs:
+        filename: 'AzCopy.exe'
+        arguments: '/Y /Source:https://onnxruntimetestdata.blob.core.windows.net/models/azcopy.exe /Dest:$(Build.BinariesDirectory)\azcopy.exe'
```

Comment:
AzCopy.exe is not available on build VMs -- how would this work? Unless AzCopy.exe is installed previously somehow.

Reply:
It is there?

Reply:
Oh.. We preinstall azcopy v8, which is outdated. Here I'm trying to update it to v10.